docs/advanced-configuration.md

```diff
@@ -103,6 +103,12 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+## Running the container in non-root user mode
+
+Collecting logs often requires reading log files that are owned by the root user. By default, the container runs with `securityContext.runAsUser = 0` which gives the `root` user permission to read those files. To run the container in `non-root` user mode, set `.Values.otelAgent.runInitForNonRoot` to `true`to cause the container to run the required file system operations as UID and GID `20000`.
+
+Note: `cri-o` container runtime did not work during internal testing.
```

Comment:
We cannot switch to this approach by default it it breaks the functionality for cri-o runtime. Please investigate further

Reply:
so this is not the default in this PR.